generate_profiles/BuildXmlFiles.groovy

```diff
@@ -2,11 +2,15 @@ import groovy.xml.MarkupBuilder;
 import FsbClassifier;
 import static FsbClassifier.*;
 @Grapes([
-    @Grab(group='com.github.spotbugs', module='spotbugs', version='3.1.0-RC2'),
+    @Grab(group='com.github.spotbugs', module='spotbugs', version='3.1.0-RC4'),
     @Grab(group='com.mebigfatguy.fb-contrib', module='fb-contrib', version='7.0.0'),
     @Grab(group='com.h3xstream.findsecbugs' , module='findsecbugs-plugin', version='1.6.0')]
 )
 
+FB = new Plugin(groupId: 'com.github.spotbugs', artifactId: 'spotbugs', version: '3.1.0-RC4')
```

Comment:
I think it's better to hold these values together as long as we are not able to specify versions as constants